java/client/build.gradle

```diff
@@ -295,7 +297,12 @@ tasks.withType(Test) {
         showStandardStreams true
     }
     // This is needed for the FFI tests
-    if (osdetector.os == 'linux' && osdetector.release.id == 'alpine') {
+    if (project.hasProperty('target')) {
+        target = project.target
+        jvmArgs "-Djava.library.path=${projectDir}/../target/${target}/release"
+    } else if (osdetector.os == 'linux' && osdetector.release.id != 'alpine') {
```

Comment:
I was concerned `jar` (#L314) and `shadowJar` (#L369) also needed to be updated to contain an additional non-alpine clause, but they seem to work without any further changes.